tensorflow_addons/layers/multihead_attention.py

```diff
@@ -70,19 +70,16 @@ class MultiHeadAttention(tf.keras.layers.Layer):
         bias_regularizer: regularizer, regularizer for the bias weights.
         bias_constraint: constraint, constraint for the bias weights.
 
-    Call Arguments
-        inputs:  List of the following tensors:
-            * `query`: Tensor of shape `(..., query_elements, query_depth)`
-            * `key`: `Tensor of shape '(..., key_elements, key_depth)`
-            * `value`: Tensor of shape `(..., key_elements, value_depth)` (optional)
+    Call Arguments:
+        inputs:  List of the tensors `[query, key, value]` where  `query` has shape `(..., query_elements, query_depth)`, `key` has shape '(..., key_elements, key_depth)`, and `value` has shape `(..., key_elements, value_depth)`. `value` is optional, if not given `key` will be used.
```

Comment:
Maybe we can leave listings `-` alone.
https://github.com/tensorflow/tensorflow/blob/v2.2.0/tensorflow/python/keras/layers/pooling.py#L183-L187
https://www.tensorflow.org/api_docs/python/tf/keras/layers/AveragePooling1D

Reply:
Yes I would prefer we stick to what's being done elsewhere in the ecosystem. I think if we copy the formatting as linked above it comes out quite readable in the generated docs. What do you think @cgarciae ?